package.json

```diff
@@ -149,7 +123,6 @@
     "@testing-library/react-hooks": "^1.1.0",
     "amphtml-validator": "^1.0.23",
     "assets-webpack-plugin": "^3.9.8",
-    "audit-ci": "2.1.0",
```

Comment:
You found an omission. This should be being used in CI, using npx in CI would be fine though.

Reply:
Added ```audit:ci``` to production tests.